WooCommerce/Classes/ViewModels/InAppPurchases/UpgradesViewModel.swift

```diff
@@ -58,6 +58,15 @@ final class UpgradesViewModel: ObservableObject {
         }
     }
 
+    /// Sync wrapper for `fetchViewData`, so can be called directly from where this
+    /// ViewModel is referenced, outside of the initializer
+    ///
+    public func retryFetch() {
+        Task {
+            await fetchViewData()
+        }
+    }
```

Comment:
We have a bit of duplication here, since this is only a wrapper for calling `fetchViewData`, but it felt adequate to duplicate the function so I didn't had to add async logic or Tasks to the view, neither deal with the state, or change the access control for the current `fetchViewData`. That said, I guess we could do just that as well and remove an extra method 🤔